frontend/src/domain/types/mission.ts

```diff
@@ -1,59 +1,67 @@
+import type { ControlUnit } from './controlUnit'
+import type { GeoJSON } from './GeoJSON'
 import type { SeaFront } from '../entities/alerts/constants'
+import type { Except } from 'type-fest'
 
-export type Mission = {
-  alertType: MissionAlertType
-  cacemNote: string
-  closedBy: string
-  cnspNote: string
+export interface Mission {
+  closedBy?: string
   controlUnits: ControlUnit[]
-  endDate: Date
-  goals: MissionGoal[]
-  hasOrder: boolean
-  id: string
-  inspectionsCount: number
-  isUnderJdp: boolean
-  openedBy: string
-  seaFront: SeaFront
-  startDate: Date
-  status: MissionStatus
-  themes: string[]
-  type: MissionType
-  zones: Record<string, any>[]
+  // We type it as `undefined` because we don't need that prop in Fish
+  envActions: undefined
```

Comment:
Indeed that would be nice but I can't use never since the strict-typing will reject [that line](https://github.com/MTES-MCT/monitorfish/blob/ivan/feat-frontend-link-mission-form-to-monitorenv-api/frontend/src/features/SideWindow/MissionForm/utils.ts#L50) whether I set an undefined value or I don't set it at all...